.gitignore

```diff
@@ -159,9 +159,10 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
 
-
 # Vim swap files
 *.swp
 .*.swp
 *~
 .*~
+.vscode/settings.json
+poetry.lock
```

Comment:
Are you sure you don't want to push `poetry.lock` to the repo? See poetry docs: https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control

Reply:
Ooh. Yeah, I think you're right. Will do!